trading/exchanges/exchange_manager.py

```diff
@@ -173,6 +174,7 @@ def _load_config_symbols_and_time_frames(self):
 
     # SYMBOLS
     def _set_config_traded_pairs(self):
+        self.cryptocurrencies_traded_pairs = {}
```

Comment:
Already in __init__, required ?

Reply:
Since it's a set, i think its better to reset the values, now its not required